src/accept.jl

```diff
@@ -0,0 +1,41 @@
+function always_accept(partition::Partition)
+    """ Accepts new partition with probability 1.
+    """
+    return 1
+end
+
+
+function pass_acceptance_fn(partition::Partition, acceptance_fn::Function)

```

Comment:
Could you specify they return type here? (```::Bool```)

Reply:
yes! excellent point